physics/mass_energy_equivalence.py

```diff
@@ -0,0 +1,41 @@
+"""
+Mass-energy equivalence
+Reference: https://en.wikipedia.org/wiki/Mass%E2%80%93energy_equivalence
+
+Here it defines a Function to calculate energy(in joules) equivalent of a given
+mass(in kilograms) using Einstein's Mass-energy equivalence equation E=mc^2
+where m is mass and c = 299,792,458 meter per second is the speed of light
+in vacuum (a constant value).
+
+The function takes a single argument mass(in kg) having datatype float and
+returns the energy(in joule) having datatype float
```

Comment:
This is a description of the function, so please put it in the function's docstring rather than the header string.